src/main/java/internet/shop/model/Car.java

```diff
@@ -0,0 +1,40 @@
+package internet.shop.model;
+
+import java.util.List;
+
+public class Car extends Driver {
```

Comment:
why does the car extends driver ?

Reply:
![image](https://user-images.githubusercontent.com/36328484/104296064-e3169600-54c9-11eb-83f0-01d38b1a8365.png)
but if it doesn't requirement I will remove it)

Reply:
IN the description it is shown as aggregation and you've used extension in the code